features/upgrade/storage/upgrade.feature

```diff
@@ -139,3 +139,102 @@ Feature: Storage upgrade tests
       | ls | /mnt/storage |
     Then the output should contain:
       | test-upgrade |
+
+  # @author chaoyang@redhat.com
+  #  @upgrade-prepare
+  #@users=upuser1,upuser2
+  @admin
+  Scenario: Snapshot operator should be in available status after upgrade and can created pod with snapshot
+    Given I switch to cluster admin pseudo user		
```

Comment:
I'd like to add a check about ocp version here, only OCP version eq 4.4.0, we can run this case.